cmd/gotagger/main.go

```diff
@@ -102,20 +105,51 @@ func (g *GoTagger) Run() int {
 		return genericErrorExitCode
 	}
 
+	zerolog.SetGlobalLevel(zerolog.Disabled)
+	if g.debug {
+		zerolog.SetGlobalLevel(zerolog.DebugLevel)
+	}
+	zerolog.TimeFieldFormat = zerolog.TimeFormatUnixMs
+	zerologr.NameFieldName = "logger"
+	zerologr.NameSeparator = "/"
+	zl := zerolog.New(zerolog.ConsoleWriter{Out: g.Stderr, TimeFormat: time.StampMicro})
+	zl = zl.With().Caller().Timestamp().Logger()
+
+	rootLogger := zerologr.New(&zl)
+	logger := rootLogger.WithName("main")
+
+	if g.configFile == defaultConfigFlag {
+		// If there's no config file provided, check for one locally.
+		defaultConfig := filepath.Join(g.WorkingDir, defaultConfigFlag)
+		logger.V(1).Info("checking for default config file")
```

Comment:
Good idea!

Reply:
I actually went with a different option. Since I used `V(1)` everywhere, I just set it once when I create / set the logger. Otherwise, I'd have to either define that const in two places or put it in some shared package.

Reply:
That works.